dev_mode/webpack.config.js

```diff
@@ -131,7 +131,7 @@ const plugins = [
     template: plib.join('templates', 'template.html'),
     title: jlab.name || 'JupyterLab'
   }),
-  new webpack.HashedModuleIdsPlugin(),
+  new webpack.ids.HashedModuleIdsPlugin(),
```

Comment:
Hashed module ids will increase code size, what’s the use case? Asking because I used to base the module federation prototype of hashed ids till Tobias showed me an alternative

Reply:
We had switched to hashed ids to enable [caching](https://webpack.js.org/guides/caching/)/